### PR TITLE
DOC-2640: Fix August Triggers and Functions changelog

### PR DESCRIPTION
**Pull request by cmilesb**
Staging link: https://docs.redis.com/staging/DOC-2640/rc/changelog/august-2023/

### CODE REVIEW

File: content/rc/changelog/august-2023.md [hunk 1]
```diff
@@ -13,8 +13,13 @@ This changelog lists new features, enhancements, and other changes added to Redi
 
 ## New features
 
-### Triggers and functions support
+### Triggers and functions preview
 
-[Triggers and functions]({{<relref "/stack/triggers-functions">}}) (previously known as RedisGears) is now available when you select Redis Stack when you [create a database]({{<relref "/rc/databases/create-database">}}) and use [Redis 7.2]({{<relref "/rc/changelog/june-2023##redis-72-opt-in">}}).
+A preview of [Triggers and functions]({{<relref "/stack/triggers-functions">}}) (previously known as RedisGears) is now available in the following regions:
```
Comment (rrelledge) [inline content/rc/changelog/august-2023.md:18]:
```suggestion
A preview of [triggers and functions]({{<relref "/stack/triggers-functions">}}) (previously known as RedisGears) is now available in the following regions:
```

**Review by rrelledge — APPROVED** [review]
Looks good to me.

File: content/rc/changelog/august-2023.md [hunk 2]
```diff
@@ -13,8 +13,13 @@ This changelog lists new features, enhancements, and other changes added to Redi
...
-### Triggers and functions support
+### Triggers and functions preview
 
-[Triggers and functions]({{<relref "/stack/triggers-functions">}}) (previously known as RedisGears) is now available when you select Redis Stack when you [create a database]({{<relref "/rc/databases/create-database">}}) and use [Redis 7.2]({{<relref "/rc/changelog/june-2023##redis-72-opt-in">}}).
+A preview of [triggers and functions]({{<relref "/stack/triggers-functions">}}) (previously known as RedisGears) is now available in the following regions:
+
+- AWS Asia Pacific - Singapore (`ap-southeast-1`)
+- GCP Asia Pacific - Tokyo (`asia-northeast1`)
+
+To use it, [create a fixed subscription]({{<relref "/rc/subscriptions/create-fixed-subscription">}}) with [Redis 7.2]({{<relref "/rc/changelog/june-2023##redis-72-opt-in">}}) and select Redis Stack when you [create your database]({{<relref "/rc/databases/create-database">}}).
```
Comment (thomascaudron) [inline content/rc/changelog/august-2023.md:23]: >  and select Redis Stack

 and select Redis Stack or Redis with the RedisGears module

**Review by thomascaudron — COMMENTED** [review]
Can we add to direct customers for the flexible subscription:

> Please reach out to support @redis.com for a private preview in a flexible subscription.

**Pull request merged**